Owners list features added

marcodarko opened the pull request:
Added owner document totals to owners list and a link to the owner's Github page.

newgene commented:
@marcodarko thanks for your pull request. Two comments before I merge it:

* are those changes on es.py needed? 
* that backup json file should not be committed.

marcodarko commented:
@newgene I deleted the end="" from the print statements from the es.py file as they kept giving me syntax errors when reading the file, from what I understand it should still give you the same result as the default behavior of print is "/n".  I removed them to finish installing the project locally, sorry I forgot to mention that change, my bad!   json file was ignored in my latest commit btw! TY!

marcodarko commented:
@newgene  you can revert it, sorry.  I commented out because or some reason some of the existing ajax calls wouldn't stop and the spinning wheel would go on forever on my computer.  I only commented out so I could continue with my work.
